### PR TITLE
Feature - add lookup table + unit test

### PR DESCRIPTION
**Pull request by lukey-luke**
## What was the problem?

- Previously the API always returned the code for asthma. Now it should return the classification mapped to the diagnostic code provided (if applicable)

Associated tickets or Slack threads:

- [Integrate lookup table in Python endpoint #160](https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/160)
- [Create unit tests for API endpoint #161](https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/161)

## How does this fix it?[^1]

- adds lookup table and BRD data as .json files
- adds python code for creating a dictionary based on the lookup table json
- updates existing python endpoint to pull from the new lookup table

## How to test this PR
- See [Unit Tests in README.md](https://github.com/department-of-veterans-affairs/abd-vro/tree/feature-unit-tests/domain-cc#unit-tests)

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: domain-cc/svc-cc-app/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+""" Pytest configuration. This file is automatically loaded by pytest before any tests. """
+
```
Comment (lukey-luke) [inline domain-cc/svc-cc-app/tests/conftest.py:2]: have no idea what's up w/ `isort` in CI here
Ran isort over my code and it exists w/ no output to console.

Screenshot 2023-05-26 at 10 59 08 AM

Comment (lukey-luke) [reply domain-cc/svc-cc-app/tests/conftest.py:2]: weird, good find! Thank you!

File: domain-cc/svc-cc-app/src/util/data/dc_lookup_table_mvp.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "5329": 9005, "6510": 9012, "6511": 9012, "6512": 9012, "6513": 9012, "6514": 9012, "6523": 9012, "6524": 9012, "6600": 9012, "6601": 9012, "6602": 9012, "6603": 9012, "6604": 9012, "6732": 6890, "6825": 9012, "6826": 9012, "6827": 9012, "6828": 9012, "6829": 9012, "6830": 9012, "6831": 9012, "6832": 9012, "6833": 9012, "6845": 9012, "6846": 9012, "6847": 9012, "7101": 8979, "8100": 9007, "9918": 8967, "7718": 8981, "7725": 8981
+}
```
Comment (msnwatson) [inline domain-cc/svc-cc-app/src/util/data/dc_lookup_table_mvp.json:3]: nit: newline at EOF

**Review by msnwatson — APPROVED** [review]
lgtm, with one nit

**Review by christiejibaraki — APPROVED** [review]
lgtm2

File: domain-cc/svc-cc-app/src/util/data/dc_lookup_table_mvp.json (hunk 2 above)
Comment (lukey-luke) [reply domain-cc/svc-cc-app/src/util/data/dc_lookup_table_mvp.json:3]: won't fix, this is programmatically generated

**Pull request merged**